pkg/ring/client/pool.go

```diff
@@ -1,8 +1,7 @@
-package client
+package grpcclient
```

Comment:
I'm wondering is that ok to have a package name different than the folder name ?

Reply:
The only include I've seen so far is renaming I'm wondering if that has an effect we you don't rename ?

Reply:
Thanks for noticing it! It wasn't intended and I've fixed it.